### PR TITLE
Add Amazon MSK event structure

### PR DESCRIPTION
**Pull request by ewbankkit**
Closes https://github.com/aws/aws-lambda-go/issues/315.

Adds Amazon MSK event structures.

```console
$ go test ./events/
ok  	github.com/aws/aws-lambda-go/events	0.020s
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: events/msk.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+package events
+
+type MskEvent struct {
```
Comment (bmoffatt) [inline events/msk.go:5]: capitalize as `MSKEvent`

File: events/msk.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	EventSource    string                 `json:"eventSource"`
+	EventSourceARN string                 `json:"eventSourceArn"`
+	Records        map[string][]MskRecord `json:"records"`
+}
+
+type MskRecord struct {
```
Comment (bmoffatt) [inline events/msk.go:11]: rename `MSKEventRecord`

File: events/msk.go [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (bmoffatt) [inline events/msk.go:1]: 2020, to match file creation date

File: events/msk_test.go [hunk 4]
```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (bmoffatt) [inline events/msk_test.go:1]: 2020, to match file creation date

**Review by bmoffatt — CHANGES_REQUESTED** [review]

**Comment by bmoffatt** [comment]
@carlzogh should we name this `KafaEvent` instead to match the java library?

ref: https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/KafkaEvent.java

**Comment by carlzogh** [comment]
> @carlzogh should we name this `KafaEvent` instead to match the java library?
> 
> ref: https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/KafkaEvent.java

Yes I think we should name it `KafkaEvent` to stay consistent.

**Comment by ewbankkit** [comment]
Renaming done.

**Review by bmoffatt — APPROVED** [review]

**Pull request merged**